src/main/java/de/drachir000/utils/config/SimpleConfigLib.java

```diff
@@ -42,4 +44,47 @@ public static Configuration buildConfiguration(String source) throws JSONExcepti
 		return new Configuration(new JSONObject(source));
 	}
 	
+	/**
+	 * Saves a {@link Configuration} to a {@link File}.
+	 *
+	 * @param configuration the {@link Configuration} to be saved
+	 * @param file          the {@link File} to save the {@link Configuration} to
+	 * @throws IOException if an I/O error occurs while writing to the file
+	 * @since 1.3
+	 */
+	public static void save(Configuration configuration, File file) throws IOException {
+		
+		BufferedWriter writer = new BufferedWriter(new FileWriter(file));
+		writer.write(configuration.toString());
+		
+		writer.flush();
+		writer.close();
+		
+	}
+	
+	/**
+	 * Reads a JSON string from a {@link File} and constructs a {@link Configuration} object from it.
+	 *
+	 * @param file the {@link File} to read the JSON string from
+	 * @return a {@link Configuration} object constructed from the JSON string
+	 * @throws IOException   if an I/O error occurs while reading the file
+	 * @throws JSONException if there is a syntax error in the JSON string or a duplicated key
+	 * @since 1.3
+	 */
+	public static Configuration load(File file) throws IOException, JSONException {
+		
+		BufferedReader reader = new BufferedReader(new FileReader(file));
+		StringBuilder builder = new StringBuilder();
+		
+		String line;
+		while ((line = reader.readLine()) != null) {
+			builder.append(line);
+		}
+		
+		reader.close();
+		
+		return buildConfiguration(builder.toString());
```

Comment:
You don't check, wether the String is blank. What would happen if it were to be blank? Would an empty JSONObject get created, or will a JSONException be thrown? What do we want here. Because if we replace a blank String with "{}" it would probably make it easier to use in the most cases.

Reply:
Implemented in https://github.com/Drachir000/SimpleConfigLib/commit/42a8f41e4f08a57fe93b688710434b6ab987648b